source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz/custom_solution_for_self_managed_user_system/quick_start_guide_service_apps.rst

```diff
@@ -1,6 +1,45 @@
+.. sectionauthor:: Narendra
+
 .. _quick_start_guide_service_apps:
 
 Quick Start Guide - Service Apps
 ================================
 
-<Content here - Owner: Narendra>
+Register Application
+++++++++++++++++++++
+
+1. Signup for a new COCO account or Sign In if you have one: https://manage.getcoco.buzz/
+
+.. image:: ../../../_static/login.png
+
+2. Click on Applications > (+) Application
+
+.. image:: ../../../_static/createApplication.png
+
+3. Select application type as **Service** and fill in the details
+   For grouping of applications, select existing app group or create a new one.
+
+.. image:: ../../../_static/registerApp.png
+
+.. image:: ../../../_static/serviceAppDetails.png
+
+5. In the app, use the above client credentials with the COCO token API to get access/refresh tokens to access COCO cloud.
```

Comment:
It in not clear how the developer copies the credentials for use since the Client Secret is shown as a series of dots ......

Reply:
This should be mentioned as a step